frontend/src/API/UserService/confirmUser.ts

```diff
@@ -0,0 +1,9 @@
+import axiosInstance from "services/axios";
+
+export const confirmUser = async (password: string) => {
+    const response = await axiosInstance.post(
+        `/user/confirm`,
+        JSON.stringify({ password: password })
+    );
+    return response;
+};
```

Comment:
This may be implicitly returned, i don't know if that would be more readable: 
```tsx
export const confirmUser = async (password: string) =>  await axiosInstance.post(
        `/user/confirm`,
        JSON.stringify({ password: password })
    );
```